### ISSUE TITLE
[Carousel] Ensure active user interface components have sufficient contrast (CAL-AMP46) 

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#13539]
**monday.com sync:** #10821963630

### Violation: 
 Ensure active user interface components have sufficient contrast 

### Recommended solution
_TBD_ - requires design consultation on the contrast between active and non-active component buttons to switch between `carousel-item`s.

<img width="445" height="210" alt="Image" src="https://github.com/user-attachments/assets/fc76d907-8ed0-4e87-9cea-5d2060b18c25" />

### WCAG Reference: 
- [1.4.11 Non-test Contrast](https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast.html) 

### Severity: 
 Low 

### Media Type: 
 Color and Contrast 

--- 
### Areas for Remediation: 
- Title: CAL-AMP46 Ensure active user interface components have sufficient contrast 
- Module: M6 Carousel, Dialog 
- Finding ID: [CAL-AMP46](https://esri.hub.essentia11y.com/short-link/S6iuDQtGVx_md3nQ) 

**Issue**
There are user interface components with a contrast ratio below 3.00:1.

Foreground color: #C9C9C9
Background color: #F7F7F7
Contrast ratio: 1.55:1

Examples of this user interface component include:
- Carousel dots (unselected state).

**User Impact**
Users with low vision will have difficulty identifying this content.

**Code Reference** 
 ```
<button role="tab" aria-selected="true" class="pagination-item pagination-item--individual pagination-item--visible pagination-item--selected" data-index="0" title="Pie chart"><calcite-icon scale="l" aria-hidden="true" icon="bullet-point-large" calcite-hydrated=""></calcite-icon></button>

```

**Steps to Reproduce**
1. Locate the user interface component.
2. Inspect it with Chrome DevTools.
3. Review its CSS to determine the foreground and background colors or select the colors of the foreground and background with a color picker.
4. Use a calculator such as https://www.levelaccess.com/color-contrast-checker-new/ to determine the contrast ratio.
5. Notice that the contrast ratio is below 3.00:1.
 
**Recommendation**
Ensure active user interface components have sufficient contrast. The required minimum contrast ratio is 3.00:1.

Common examples of qualifying components include text field borders, check marks for checkboxes, fillings for radio buttons, focus indicators, and icon-only controls. Non-interactive controls are exempt from this requirement.

To calculate color contrast ratios, use a tool such as the Level Access Accessible Color Picker Chrome extension: https://chrome.google.com/webstore/detail/accessible-color-picker/bgfhbflmeekopanooidljpnmnljdihld or the Color Contrast Checker: https://www.levelaccess.com/color-contrast-checker-new/

**Compliant Code Example**
```
<!-- Example 1: A Send button (with/without a visual boundary) has a focus indicator when focused. The
focus indicator for the component meets the sufficient contrast when the button is focused. -->
---
<!-- Example 2: Visual focus indicator for a link that is a blue (#6699cc) outline around the link. The blue (#6699cc) outline does provide a sufficient contrast with the white (#ffffff) background that is equal to 3 to 1.-->
---
<!-- Example 3: A Submit button at the bottom of a form that is visible, but cannot be activated until all
the required fields in the form are completed does not meet the sufficient color contrast requirement (a disabled control in HTML are not required to meet requirements). -->
```

 **Issue Image** 
 https://desktopqa/aic_lap/Calcite/Calcite2025/CAL-AMP46.png 
 

 --- 
### Additional Resources: 
- [LA Platform Report Link](https://esri.hub.essentia11y.com/short-link/S6iuDQtGVx_md3nQ) 
- [Esri Accessibility Resources](https://esriis.sharepoint.com/sites/Accessibility)
- Report Source: Product accessibility evaluation conducted on Calcite by Level Access and delivered August, 2025.
